packages/react/src/components/Grid/Column.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings } from 'carbon-components';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const { prefix } = settings;
+
+function Column({
+  as: BaseComponent = 'div',
+  children,
+  className: containerClassName,
+  sm,
+  md,
+  lg,
+  xlg,
+  max,
+  ...rest
+}) {
+  const columnClassName = getClassNameForBreakpoints([sm, md, lg, xlg, max]);
+  const className = cx(containerClassName, columnClassName, {
+    [`${prefix}--col`]: columnClassName.length === 0,
```

Comment:
shouldn't the `col` class always be applied like in `Grid` and `Row`?
```suggestion
    [`${prefix}--col`]: true,
```

Reply:
@ZacharyStair I believe all of the grid column classes are setup this way and `bx--col` is a short-hand for the auto-span!